arcgis-ios-sdk-samples/Content Display Logic/PortalItems.plist

```diff
@@ -77,6 +77,10 @@
 			<string>SA_EVI_8Day_03May20.zip</string>
 		</dict>
 		<dict>
+			<key>filename</key>
+			<string>PSCC_vector.vtpk</string>
+			<key>identifier</key>
+			<string>aae86ca2cf8f4e6087065fb8bf1b6ecd</string>
```

Comment:
These should be in their own dictionary. As part of this one, this dictionary now has multiple values for the keys `filename` and `identifier`. In the plist editor, the last one read wins out:

![image](https://user-images.githubusercontent.com/2257493/96899546-e576d000-1445-11eb-8368-9f3e38067406.png)